ghwatch.py

```diff
@@ -262,6 +276,9 @@ def draw():
 
     return buttons
 
+def priority_sort_key(item):
```

Comment:
i'd prefer to move this function above `draw()` (so that it's defined by the time it's called, this makes the code more readable for C-ers :smile: )